caravel/assets/visualizations/big_number.js

```diff
@@ -6,6 +6,7 @@ require('./big_number.css');
 
 var px = require('../javascripts/modules/caravel.js');
 
+
```

Comment:
two spaces breaks one of our JS linting rules, I don't think it's well-documented but you should run `npm run lint` before PRs from the `assets/` folder (can also add to sublime if you use that, it will flag them as you type). The travis build I think should catch it but looks like it's failing on other npm errors.